app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>All Posts will be shows here</h1>
\ No newline at end of file
+<div>
+  <div>
+        <%= image_tag @posts[0].author.photo, alt: "User pic" %>
+         <div>
+           <h1><%= @posts[0].author.name %></h1>
+           <p>Number of posts: <%= @posts[0].author.posts_counter %></p>
+         </div>
+  </div>
+
+
+<% if @posts.empty? %> 
+  <div>This user has not published any post yet!</div>
+<% else %>
+<% @posts.each_with_index do |post, index| %>
+
+  <div>
+    <div><%= link_to "Post  ##{index + 1}", user_post_path(post.author, post) %></div>
+    <br>
+    <div><%= post.text %></div>
+    <div>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></div>
+  </div>
+
+
+   <% if post.five_most_recent_comments == [] %> 
+     <p>This post has no comments yet!</p>
+   <% else %>
+     <% post.five_most_recent_comments.each do |comment| %>
+       <ul>
+         <li><%= post.author.name %>: <%= comment.text %></li>
+       </ul>
+     <% end %>
```

Comment:
- [ ] Same thing here as well the `five_most_recent_comments` is undefined I believe you're trying to access the `recent_5_comments` instead so kindly fix this issue.
### Note for future reviewers 📝  
- [ ] Since I wasn't able to open the posts index page please make sure the layout matches the one shown in the [wireframe here](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACJTIBN7WB3L7SZBUVMDGHVYZKE5GXA) and the page functions properly.